### PR TITLE
Implement endpoint that surface function baseline to clients

### PR DESCRIPTION
**Pull request by cyy0714**

### CODE REVIEW

File: thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/AnomalyResource.java [hunk 1]
```diff
@@ -508,6 +512,68 @@ public Response applyAutotuneConfig(@PathParam("autotuneConfigId") @NotNull long
...
+   *      the window size of the monitoring window, ex: 1d, 6h, and so on
+   * @return
+   * @throws Exception
+   */
+  @GET
+  @Path(value = "/anomaly-function/{id}/baseline")
+  @Produces(MediaType.APPLICATION_JSON)
+  public Map<String, AnomalyTimelinesView> getTimeSeriesAndBaselineData(@PathParam("id") long functionId,
+      @QueryParam("start") String startTimeIso, @QueryParam("end") String endTimeIso,
+      @QueryParam("monitoringWindow") String monitoringWindow) throws Exception {
```
Comment (bl44) [inline thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/AnomalyResource.java]: it seems to me that the input: monitoringWindow is not used in your function

Comment (cyy0714) [inline thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/AnomalyResource.java]: Removed. Thanks for the comment!

File: thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/AnomalyResource.java [hunk 2]
```diff
@@ -508,6 +512,66 @@ public Response applyAutotuneConfig(@PathParam("autotuneConfigId") @NotNull long
...
+   *
+   * The baseline is generated from getTimeSeriesView() in each detection function.
+   * @param functionId
+   *      the target anomaly function
+   * @param startTimeIso
+   *      the start time of the monitoring window (inclusive)
+   * @param endTimeIso
+   *      the end time of the monitoring window (exclusive)
+   * @return
+   *      the Map of AnomalyTimelinesView
```
Comment (cyenjung) [inline thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/AnomalyResource.java]: Need to specify what is the meaning of the String of the map.

Comment (cyy0714) [inline thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/AnomalyResource.java]: Done

**Pull request merged**